poi-ooxml/src/main/java/org/apache/poi/xssf/streaming/SXSSFSheet.java

```diff
@@ -61,11 +62,7 @@ protected SXSSFSheet(SXSSFWorkbook workbook, XSSFSheet xSheet, int randomAccessW
         _sh = xSheet;
         calculateLeftAndRightMostColumns(xSheet);
         setRandomAccessWindowSize(randomAccessWindowSize);
-        try {
-            _autoSizeColumnTracker = new AutoSizeColumnTracker(this);
-        } catch (UnsatisfiedLinkError | InternalError e) {
-            LOG.atWarn().log("Failed to create AutoSizeColumnTracker, possibly due to fonts not being installed in your OS", e);
-        }
+        _autoSizeColumnTracker = new AutoSizeColumnTracker(this);
```

Comment:
please do not randomly change code like this- undo this change or the PR is rejected